src/rust-2021/reserving-syntax.md

```diff
@@ -2,44 +2,47 @@
 
 ## Summary
 
-- `any_prefix#..`, `any_prefix".."`, and `any_prefix'..'` are reserved syntax, and no longer tokenize.
+- `any_prefix#`, `any_prefix"..."`, and `any_prefix'...'` are now reserved
```

Comment:
I'm not sure if the use of `any_syntax` is clear. Perhaps we describe it in words (e.g., "the use of any syntax followed by a "#" or literals enclosed in single or double quotes (e.g., `foo#`, `bar'baz'` , `qux"corge"`) is now reserved")

Reply:
I agree that we should at least define `any_prefix`

Reply:
I've changed the line in the summary to make it clear that this is an identifier. The subsequent non-summary explanation goes into more detail on that point, so I think this is now addressed.